fractal/client/authclient.py

```diff
@@ -39,7 +40,13 @@ async def _get_fresh_token(self):
             )
         raw_token = res.json()
         self.token = raw_token["access_token"]
-        with open(Path(settings.SESSION_CACHE_PATH).expanduser(), "w") as f:
+
+        # Create cache folder, if needed
+        cache_dir = str(Path(f"{settings.FRACTAL_CACHE_PATH}").expanduser())
+        if not os.path.isdir(cache_dir):
```

Comment:
Ok for this instance.

FYI, cf https://docs.python.org/3/library/pathlib.html?highlight=pathlib%20path#pathlib.Path.mkdir

Reply:
> Rather than completely changing `*_id (int)` with `*_name (str)` it would have probably been better to add a compatibility layer, e.g., try cast to int, else it is a name.

This is done

Reply:
See https://github.com/fractal-analytics-platform/fractal/pull/273/commits/ba5688a7ba09d612de61f617980716acb32071a3